src/app/components/turnos/gestor-agendas/gestor-agendas.scss

```
@@ -18,3 +18,23 @@ span.badge,
 span.badge-default {
     font-size: 0.6rem;
 }
+table{
```

```suggestion
.table-lista-agendas {
```
- Es mejor no poner una regla para un elemento global (table), en su lugar aplicar una clase, porque si alguna vez se agrega otra tabla al componente, la misma va a tomar las reglas globales también.
- Las demás reglas de la tabla pueden ir anidadas, quedando: 
`.table-lista-agendas {
    height: 100%;

    tbody {
        display: block;
        height: 100%;
        overflow: scroll;
    }

    thead,
    tbody tr {
        display: table;
        width: 100%;
        table-layout: fixed;
    }
}`